src/include/gurt/atomic.h

```diff
@@ -43,7 +43,18 @@
 #if HAVE_STDATOMIC
 
 #include <stdatomic.h>
+
+
+#if defined(__cplusplus)
```

Comment:
What has __cplusplus got to do with it?  If the header file needs this guard in place then it should be in it's own PR with an explanation of what value it brings.

Reply:
I am not sure if we need this one. the atomic built-ins also have different prototypes for C and C++ code.